controllers/tasks.js

```diff
@@ -20,6 +20,15 @@ const addNewTask = async (req, res) => {
       ...req.body,
       createdBy,
     };
+
+    // Retrieve the issue assignee's RDS user info
+    if (Object.keys(body).includes("github")) {
+      if (Object.keys(body.github.issue).includes("assignee")) {
+        const user = await getRdsUserInfoByGitHubUsername(body.github.issue.assignee);
+        body.github.issue.assigneeRdsInfo = user;
+      }
```

Comment:
I thought we were only going to the store only id here.

Reply:
Yes, made the changes.